app/src/main/res/values/strings.xml

```diff
@@ -4,6 +4,7 @@
 
     <!--General Strings-->
     <string name="action_about_us">About Us</string>
+    <string name="action_confirm_log_out">LOG OUT</string>
```

Comment:
No need for explicit caps

Reply:
@iamareebjamal Removed the explicit caps. Please review.